extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -76,16 +82,43 @@ class AmpInstagram extends AMP.BaseElement {
   }
 
   /** @override */
-  prerenderAllowed() {
-    return true;
+  prerenderCallback() {
+    const image = new Image();
```

Comment:
What if we used `<amp-img placeholder>`? We then take care of both issues, right?